eg/hygrometer-htu21d.js

```diff
@@ -0,0 +1,16 @@
+var five = require("../lib/johnny-five.js");
+var board = new five.Board();
+
+board.on("ready", function() {
+  var hygrometer = new five.Hygrometer({
+    controller: "HTU21D"
+  });
+
+  hygrometer.on("data", function() {
+    console.log(this.relativeHumidity + " %");
```

Comment:
Do these sensors produce relative humidity and absolute humidity?

I found this useful explanation: 

- **Absolute Humidity**, often just referred to as 'the humidity', is a measure of the actual amount of water vapour in a particular sample of air: measured as a partial pressure (vapour pressure/hPa or millibars); a mixing ratio (gm water vapour/kg of dry air), dew point etc.
- **Relative Humidity** - expressed commonly as a percentage value, is the ratio of the actual amount of water vapour present in a sample (the Absolute Humidity) to that amount that would be needed to saturate that particular sample.